src/Umbraco.Cms.Api.Delivery/Filters/SwaggerDocumentationFilterBase.cs

```diff
@@ -60,19 +104,52 @@ protected void AddQueryParameterDocumentation(OpenApiParameter parameter, Dictio
             }
         });
 
-    protected void AddApiKey(OpenApiOperation operation) =>
-        operation.Parameters.Add(new OpenApiParameter
+    private void AddExpand(OpenApiOperation operation)
+        => operation.Parameters.Add(new OpenApiParameter
         {
-            Name = "Api-Key",
-            In = ParameterLocation.Header,
+            Name = "expand",
+            In = ParameterLocation.Query,
             Required = false,
-            Description = "API key specified through configuration to authorize access to the API.",
-            Schema = new OpenApiSchema { Type = "string" }
+            Description = QueryParameterDescription("Defines the properties that should be expanded in the response"),
+            Schema = new OpenApiSchema { Type = "string" },
+            Examples = new Dictionary<string, OpenApiExample>
+            {
+                { "Expand none", new OpenApiExample { Value = new OpenApiString(string.Empty) } },
+                { "Expand all properties", new OpenApiExample { Value = new OpenApiString("properties[$all]") } },
+                {
+                    "Expand specific property",
+                    new OpenApiExample { Value = new OpenApiString("properties[alias1]") }
+                },
+                {
+                    "Expand specific properties",
+                    new OpenApiExample { Value = new OpenApiString("properties[alias1,alias2]") }
+                },
+                {
+                    "Expand nested properties",
+                    new OpenApiExample { Value = new OpenApiString("properties[alias1[properties[nestedAlias1,nestedAlias2]]]") }
+                }
+            }
         });
 
-    protected string PaginationDescription(bool skip, string itemType)
-        => $"Specifies the number of found {itemType} items to {(skip ? "skip" : "take")}. Use this to control pagination of the response.";
-
-    private string QueryParameterDescription(string description)
-        => $"{description}. Refer to [the documentation]({DocumentationLink}#query-parameters) for more details on this.";
+    private void AddFields(OpenApiOperation operation)
+        => operation.Parameters.Add(new OpenApiParameter
+        {
+            Name = "fields",
+            In = ParameterLocation.Query,
+            Required = false,
+            Description = QueryParameterDescription("Explicitly defines which properties that should be included in the response (by default all properties are included)"),
+            Schema = new OpenApiSchema { Type = "string" },
+            Examples = new Dictionary<string, OpenApiExample>
+            {
+                { "Include all properties", new OpenApiExample { Value = new OpenApiString("properties[$all]") } },
+                {
+                    "Include only specific properties",
+                    new OpenApiExample { Value = new OpenApiString("properties[alias1,alias2]") }
+                },
```

Comment:
Adding an example with a single property, to align with the `expand` examples
```suggestion
                {
                    "Include only specific property",
                    new OpenApiExample { Value = new OpenApiString("properties[alias1]") }
                },
                {
                    "Include only specific properties",
                    new OpenApiExample { Value = new OpenApiString("properties[alias1,alias2]") }
                },
```